style.css

```diff
@@ -1,7 +1,7 @@
 :root{
-    --primary-color:#ee2020;
+    --primary-color:red;
     --secondary-color:rgb(245, 245, 245);
-    --bg-color:rgba(54, 51, 51, 0.575)
+    --bg-color:rgba(37, 37, 37, 0.589);
 }
```

Comment:
Use only one way to keep it consistent.

Reply:
In order to increase the contrast of Text I had to change the background colour